gui/cefdisplay/inc/gui_handler.h

```diff
@@ -29,7 +29,9 @@ class GuiHandler : public BaseHandler, public CefDisplayHandler {
    // CefDisplayHandler methods:
    virtual void OnTitleChange(CefRefPtr<CefBrowser> browser, const CefString &title) OVERRIDE;
 
-   virtual bool OnConsoleMessage(CefRefPtr<CefBrowser> browser, const CefString &message, const CefString &source,
+   virtual bool OnConsoleMessage(CefRefPtr<CefBrowser> browser,
+                                 cef_log_severity_t level,
+                                 const CefString &message, const CefString &source,
```

Comment:
Is this from `clang-format`?

Reply:
> Is this from clang-format?

No, I did not apply clang-format for these files.